pkgs/ipxe/default.nix

```diff
@@ -1,4 +1,15 @@
-{ stdenv, writeText, perl, xz, openssl, src, lib }:
+{
+  efi ? true,
+  legacy ? true,
```

Comment:
Do these need to be parameters? What would be a reason to ever disable legacy/efi build?

Reply:
The only reason I can think of is compile time when you're building it yourself. I thought it's nice to have some configureability.

Reply:
Fair enough.